sync/config/dashboard.yaml

```diff
@@ -0,0 +1,35 @@
+# Each YAML file under sync/ configures how helper/helper.py synchronizes
+# contents of various versions from its source of truth (usually a GitHub
+# repository of a Tekton component, such as tektoncd/pipelines) to
+# content/ (for the lastest version) and vault/ (for earlier versions).
+
+# The name of the component.
+# helper.py will use this value to build directories in content/ and vault/.
+component: Dashboard
+# The order of the component.
+displayOrder: 3
+# The GitHub repository where documentation resides.
+repository: https://github.com/tektoncd/dashboard
+# The directory in the GitHub repository where contents reside.
+docDirectory: docs
+# The tags (versions) of contents to sync.
+# Note that helper.py and related script reads tags in the order specified in
+# the following list; the first entry in tags will automatically become the
+# latest version of contents.
+tags:
+  # The name of the tag in the GitHub repository.
+- name: master
```

Comment:
Should we pull the latest released version instead of master here?

Reply:
We're pulling master for the other projects already so I went for consistency here. It looks like there's some rudimentary support for multiple versions baked in but it just links to the project repo rather than hosting the versioned docs on the website.

We should discuss this at the docs working group and maybe the broader group too to agree a proper strategy.

Also related: https://github.com/tektoncd/website/issues/88 and https://github.com/tektoncd/website/issues/34

Reply:
Forgot to add this after the call...

Discussed this at the docs working group on April 22nd. With our current approach, switching the sync script to pull specific release tags instead of master means we wouldn't be able to publish updated docs between releases.

Options:
- patch version releases for doc updates if needed
- change sync script to be more flexible (what would the strategy be?)
- ???

Also discussed current support for multiple versions:
- Items under 'Previous Releases' not obviously clickable at first glance
- clicking 'Archived Versions' brings user to the project repo's 'Tags' page, not obvious how to get to desired docs from there unless familiar with the project structure (select relevant tag, find docs folder, find correct file)

Reply:
@afrittoli @AlanGreene I think will give us the flexibility we want.

Reply:
updated to pull v0.6.1.4 as the default, with `master` listed under previous releases